### PR TITLE
Feature/vue 3

### PR DESCRIPTION
**Pull request by Startouf**
resolves https://github.com/LesGars/WatchWithMe/issues/513

### CODE REVIEW

**Comment by Startouf** [comment]
Screenshot 2020-11-15 at 20 19 00

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -19,13 +19,15 @@ jobs:
...
         run: cp extension/env/common.js.example extension/env/common.js
-      - uses: pnpm/action-setup@v1.2.0
+      - uses: pnpm/action-setup@v1.2.1
         with:
           version: 5.2.6
       - name: Install dependencies
         run: make install
-      - name: Install AWS Cli
-        uses: chrislennon/action-aws-cli@v1.1
+      - id: install-aws-cli
```
Comment (gjgd) [inline .github/workflows/cd.yml:27]: That's funny, I worked on the same problem today in a different repo: https://github.com/transmute-industries/sidetree.js/pull/121/files

Solved it by using the offical aws action: https://github.com/aws-actions/configure-aws-credentials

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -16,8 +16,10 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - name: Use default env variables
         run: cp extension/env/common.js.example extension/env/common.js
-      - name: Install AWS Cli
-        uses: chrislennon/action-aws-cli@v1.1
+      - id: install-aws-cli
```
Comment (gjgd) [inline .github/workflows/ci.yml:19]: See other comment

File: extension/env/development.env.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
 var merge = require("webpack-merge");
 var prodEnv = require("./production.env");
-var common = require("./common").dev;
+try {
+    var common = require("./common").dev;
```
Comment (gjgd) [inline extension/env/development.env.js:4]: mmmh, this looks weird, why are we wrapping a require in try catch?

Comment (Startouf) [reply extension/env/development.env.js:4]: Ah, I actually wanted a workaround so the project would still compile with default settings without adding a custom common.js ENV file to the project, but I agree I don't like this code... should I just revert or do you have suggestions ?

Comment (gjgd) [reply extension/env/development.env.js:4]: I don't know enough about this particular config to suggest an improvement, I would have to look into it. If it works as is let's merge

**Pull request merged**